docs/source/api/deliveryservices_id_routing.rst

```diff
@@ -21,12 +21,17 @@
 
 ``GET``
 =======
-Retrieves routing method statistics for a particular :term:`Delivery Service`
+Retrieves the aggregated routing percentages for a given :term:`Delivery Service`. This is accomplished by polling stats from all online Traffic Routers via the ``/crs/stats`` route.
```

Comment:
Can you link that to the TR API docs?

I thought there was already a `tr-api-crs-stats` label, but it looks like there isn't, so I guess it's not a big deal if you don't want to add it, since it's not actually related to the code changes you made.

Reply:
🎉